e2e/env/oci.go

```diff
@@ -66,12 +66,13 @@ func (c ctx) ociSingularityEnv(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		testEnv := append(os.Environ(), tt.env...)
```

Comment:
How did the absence of this (and passing just `tt.env`) not cause problems before...? 🤔 I presume that means we don't have code that tests for this kind of env-var additivity?

Reply:
Previously, we were setting an empty environment for the singularityenv, without the host `PATH`. We need that so that `squashfuse` can be found.